docs/howtos/layers/shapes.md

```diff
@@ -283,15 +251,29 @@ are used. i.e. You can't remove a vertex before you have created a shape.
   coordinates of the first endpoint and the second click marks the coordinates
   of the second endpoint. Add other lines as needed.
 
+* **Add polylines**
+  ![image: Add polyline](../../_static/images/shape-add-polyline.png)
+
+  Select the `Add polyline` tool from the layer controls panel or by pressing the
+  `shift + L` keys when the shapes layer is selected. Click where you want the polyline to
+  start and then click each location where the direction of the polyline changes,
+  this adds a vertex at that location. When you have drawn the complete path,
+  hit `Esc` or double-click. This adds a final vertex at the current mouse position and
+  completes the path. You can then add another polyline.
+
 * **Add path**
   ![image: Add path](../../_static/images/shape-add-path.png)
 
   Select the `Add path` tool from the layer controls panel or by pressing the
-  `t` key when the shapes layer is selected. Click where you want the path to
-  start and then click each location where the direction of the path changes,
-  this adds a vertex at that location. When you have drawn the complete path,
-  hit `Esc`. This adds a final vertex at the current mouse position and
-  completes the path. You can then add another path.
+  `t` key when the shapes layer is selected. The tool can be used to draw freeform, 
+  complex paths using the mouse or tablet. Importantly, the paths will still consist 
+  of vertexes and will be editable like all other Shapes. Note that for the initial 
+  drawing, sometimes there is a delay due to `numba`.
+
```

Comment:
```suggestion
  of vertexes and will be editable like all other Shapes.

```
Let's delete this. the numba thing affects all tools on shapes and it should be really rare. No sense in focusing on it here with this tool.